### DISCUSSION TITLE
Retriggerable Timer for Relay Output

### DISCUSSION THREAD

**Discussion by dgoadby** [arendst/Tasmota#20103] [Q&A] [answered]
I am using Sonoff R26's. I can turn the switch on or off using MQTT and it works fine.

The R26's are connected to some heavy loads and they should only be ON when the electricity cost is at its lowest. 

What I want to do is build-in a safeguard if any part of the system ie Server PC, network, Wifi etc fails.

I want to set the output ON and, if another ON command is not received within a time period eg 5 minutes, then the relay output will be set OFF. In effect I want the software equivalent of a retriggerable monostable.

There is a PulseTime<x> command which would time out but does not say if I can reset the time. 

Does resending a PulseTime<x> command reset the time period without having to actually time out? If it does then I'm sorted, if not then how can I create the functionality that I want?

**Comment by barbudor** [comment, marked as answer]
YES, when using Pulsetime the timer is retriggered at every ON received

**Reply by sfromis** [reply]
While resending the PulseTime command will also restart the period, this is not the way it is meant to be used.
